static/src/stylesheets/module/site-messages/_fiv-banner.scss

```diff
@@ -167,24 +187,28 @@
     }
 }
 
-.fiv-banner__background {
+.fiv-banner__background_desktop {
     position: absolute;
     top: 1px;
     right: -40px;
     width: 436px;
     height: 357px;
 }
 
-.fiv-banner__circles {
+.fiv-banner__background_mobile {
     position: absolute;
+    bottom: 1px;
+    right: -8px;
+    width: 291px;
+    height: 145px;
 }
 
 .fiv-banner__close {
-    float: left;
+    right: 0;
 
     // we want the close button above the link overlay
     // z-index only applies to positioned elements
-    position: relative;
+    position: absolute;
```

Comment:
this has actually made the desktop close icon move to the right (contrary to the design), but it was overlayed with the "G" before which was even worse.